### ISSUE TITLE
Conversion issue in SmartTV-AGH.txt?

### ISSUE THREAD

**Issue by monzta** [Perflyst/PiHoleBlocklist#52]
Shouldn't https://github.com/Perflyst/PiHoleBlocklist/blob/3c538918198b3bae2034ae07061c3345e4ac3061/SmartTV-AGH.txt#L120 and https://github.com/Perflyst/PiHoleBlocklist/blob/3c538918198b3bae2034ae07061c3345e4ac3061/SmartTV-AGH.txt#L121 rather be 

`||samsungcloudsolution.com^` and `||samsungcloudsolution.net^` (`||` instead of `|` like all the other ones)?

**Comment by monzta** [comment]
I see some `samsungcloudsolution.com` and `samsungcloudsolution.net` addresses are commented out.  Maybe that's why they are written like this?

**Comment by Perflyst** [comment]
@DandelionSprout can you check this, please?

**Comment by DandelionSprout** [comment]
When I initially created the AGH list version, I decided to do a very conservative take on the Samsung domains due to the vast number of them, especially the `samsungcloudsolution` subdomains.

So I took a qualified guess that they were listed individually in Perflyst's list for a reason (beyond just how Pi-hole required all individual subdomains), and that something like `||samsungcloudsolution.*^` would cause heavy breakage in ways we couldn't even imagine, even with around 10 whitelisted subdomains. As such I used one `|` to make sure it only blocked exactly `samsungcloudsolution.com/.net`.

That being said, if anyone here have enough knowledge about Samsung TVs to compress and distill those entries, that'd be pretty great.
